docs/CalicoSwarm.md

```diff
@@ -0,0 +1,152 @@
+# Running calico-docker networking on a Docker Swarm.
+The following instructions configure a Docker Swarm cluster networked using Calico.  In this tutorial, we will do the following.
+- Configure Docker Swarm on a VM cluster.
+- Configure etcd and Calico on our cluster.
+- Provision workloads on the Swarm, and check they can communicate.
+
+## Prerequisites
+To complete this demo, make sure you have the following prerequisites.
+- Four Linux servers (VMs or bare-metal) with Docker 1.4 or later installed.  The servers should have IP connectivity to each other.
+  - 1 client (to control the Swarm)
+  - 1 Swarm manager (to manage the Swarm nodes)
+  - 2 Swarm nodes (to host our containers)
+
+## Installing Swarm on your cluster
+Our Docker Swarm will consist of 2 nodes and a Swarm manager.  We will use the fourth server as the client to control our Swarm.  We'll use the token-based discovery backend to configure our Swarm.
+
+First, let us create the token which will identify our swarm.  To do this, run the following on your client node:
+```
+docker pull swarm
```

Comment:
Might be worth indicating that sudo could be required for the docker commands.  Depends on your docker installation.